dio_test/lib/src/test/timeout_tests.dart

```diff
@@ -18,7 +20,7 @@ void timeoutTests(
       test('with response', () async {
         dio.options.connectTimeout = Duration(milliseconds: 3);
         await expectLater(
-          dio.get('/'),
```

Comment:
Not sure about this, there is not gonna be a response so this actually tests something different now.

Reply:
The name got a minor update.